ui/src/context/Application.jsx

```diff
@@ -34,147 +30,106 @@ export { walletP };
 
 export const ApplicationContext = createContext();
 
-export function useApplicationContext() {
-  return useContext(ApplicationContext);
-}
+export const useApplicationContext = () => useContext(ApplicationContext);
 
-const watchPool = async (dispatch, ammAPI, brand) => {
-  const updatePoolState = value => {
-    console.log('updating pool state', brand, value);
-    dispatch(setPoolState({ brand, value }));
-  };
-  const poolUpdater = async () => {
-    const poolSubscription = await E(ammAPI).getPoolMetrics(brand);
-    const poolNotifier = makeNotifierFromAsyncIterable(poolSubscription);
-    for await (const value of iterateNotifier(poolNotifier)) {
-      updatePoolState(value);
+const watchOffers = dispatch => {
+  async function offersUpdater() {
+    const offerNotifier = E(walletP).getOffersNotifier();
+    for await (const offers of iterateNotifier(offerNotifier)) {
+      console.log('======== OFFERS', offers);
+      dispatch(updateOffers(offers));
     }
-  };
-  poolUpdater().catch(err =>
-    console.error('Brand watcher exception', brand, err),
-  );
+  }
+  offersUpdater().catch(err => console.error('Offers watcher exception', err));
+};
+
+const watchPoolMetrics = async (dispatch, brand, i) => {
+  const f = E(walletP).makeFollower(`:published.amm.pool${i}.metrics`);
+  for await (const { value: state } of iterateLatest(f)) {
+    dispatch(setPoolState({ brand, value: state }));
+    if (i === 0) {
+      dispatch(setCentral(state.centralAmount.brand));
+    }
+  }
 };
 
-const fetchLiquidityBrand = async (poolBrand, dispatch, ammApi) => {
-  const { Liquidity } = await E(ammApi).getPoolAllocation(poolBrand);
-  if (Liquidity?.brand) {
+const watchPoolInit = async (dispatch, brand, i, leader) => {
+  const f = makeFollower(`:published.amm.pool${i}.init`, leader, {
+    unserializer,
+  });
+
+  for await (const { value: state } of iterateLatest(f)) {
```

Comment:
`init` publishes once ever, so don't set up a watch, just read the first value.

Reply:
Done

Reply:
I had trouble getting it to work with `next()`

Reply:
Ok, so that's tech debt then. Best to mark it with `TODO` but I won't block on that.

Reply:
Added `TODO`